src/datadoc/frontend/callbacks/variables.py

```diff
@@ -244,6 +244,10 @@ def get_corresponding_identifier(
             return VariableIdentifiers.TEMPORALITY_TYPE
         case DatasetIdentifiers.DATA_SOURCE:
             return VariableIdentifiers.DATA_SOURCE
+        case DatasetIdentifiers.CONTAINS_DATA_FROM:
+            return VariableIdentifiers.CONTAINS_DATA_FROM
+        case DatasetIdentifiers.CONTAINS_DATA_UNTIL:
```

Comment:
We should implement this method as a dict, ref: https://docs.astral.sh/ruff/rules/too-many-branches/